src/EditorFeatures/CSharpTest2/Recommendations/AsyncKeywordRecommenderTests.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
+    [Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
IDE doesn't really use class traits.  Can you keep this on the individual function?